packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 49.4.0 - August 6, 2020
+
+## Added
+
+#### salesforcedx-vscode-soql
+
+- Added SOQL Builder Package for `.soql` files ([PR #2382](https://github.com/forcedotcom/salesforcedx-vscode/pull/2382))
```

Comment:
We should remove this since the soql extension will not be published for a while